docs/siem/siem-ui.asciidoc

```diff
@@ -1,102 +1,169 @@
-You can drill down for deeper insights, and drag and drop items of interest from
-the Hosts page to Timeline for further investigation.
+* A detection engine to identify attacks and system misconfigurations
+* A workspace for event triage and investigations
+* Interactive visualizations to investigate process relationships
+* Inbuilt case management with automated actions
+* Detection of signatureless attacks with prebuilt machine learning anomaly jobs
+and detection rules
 
-[role="screenshot"]
-image::siem/images/hosts-ui.png[]
-
-
-[float]
-[[network-ui]]
-=== Network
-
-The Network page displays key network activity metrics in an interactive map,
-and provides network event tables that enable interaction with Timeline.
-
-[role="screenshot"]
-image::siem/images/network-ui.png[]
-
-[float]
-[[detections-ui]]
-=== Detections (beta)
-
-The Detections feature automatically searches for threats and creates 
-alerts when they are detected. Detection rules define the conditions 
-for when alerts are created. Elastic Security comes with prebuilt rules that
-search for suspicious activity on your network and hosts. Additionally, you can
-create your own rules.
+[discrete]
+== Elastic Security components and workflow
 
-See {security-guide}/detection-engine-overview.html[Detections] for information
-on managing detection rules and alerts.
+The following diagram provides a comprehensive illustration of the Elastic Security workflow.
 
 [role="screenshot"]
-image::siem/images/detections-ui.png[]
-
-[float]
-[[cases-ui]]
-=== Cases (beta)
-
-Cases are used to open and track security issues directly in Elastic Security. 
-Cases list the original reporter and all users who contribute to a case
-(`participants`). Case comments support Markdown syntax, and allow linking to
-saved Timelines. Additionally, you can send cases to external systems from
-within Elastic Security.
-
-For information about opening, updating, and closing cases, see
-{security-guide}/cases-overview.html[Cases] in the Elastic Security Guide.
-
-[role="screenshot"]
-image::siem/images/cases-ui.png[]
-
-[float]
-[[timelines-ui]]
-=== Timeline
-
-Timeline is your workspace for threat hunting and alert investigations. 
-
-[role="screenshot"]
-image::siem/images/timeline-ui.png[Elastic Security Timeline]
-
-You can drag objects of interest into the Timeline Event Viewer to create
-exactly the query filter you need. You can drag items from table widgets within
-Hosts and Network pages, or even from within Timeline itself.
-
-A timeline is responsive and persists as you move through Elastic Security
-collecting data. 
+image::../siem/images/workflow.png[Elastic Security workflow]
+
+Here's an overview of the flow and its components:
+
+* Data is shipped from your hosts to {es} via beat modules and the Elastic
+Endpoint Security agent integration:
+** {security-guide}/install-endpoint.html[{es-sec-endpoint}] - Elastic agent integration that
```

Comment:
Only flagging because I'm not sure why the `{es-sec-endpoint}` attribute is showing up as `{es-sec-endpoint}` in this sentence instead of `Endpoint Security`.

Reply:
I think it needs to be one of the ones listed here: https://github.com/elastic/docs/blob/6367efdfd6ebbb1787e289f5fee67ba870e14c28/shared/attributes.asciidoc#L210-L212